src/main/java/com/fundation/webservice/model/KeyFrameVideo.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+/**
+ * Implements KeyframeVideo class an convert Method.
+ *
+ * @author Josue Rodriguez
+ * @version 1.0
+ */
+public class KeyFrameVideo {
+    CriteriaKeyFrameVideo criteria;
+
+    public KeyFrameVideo(CriteriaKeyFrameVideo criteria) {
+        this.criteria = criteria;
+    }
+
+    //The convert method add a keyframe for every certain amount of frame.
+    public void convert() {
+        try {
+            String cmd = "ffmpeg -i "
+                    + criteria.getSrcPath()
+                    + " -vcodec libx264 -x264-params keyint="
+                    + criteria.getFrames()
+                    + ":scenecut=0 -acodec copy "
+                    + criteria.getDestPath()
+                    + criteria.getName() + "."
+                    + criteria.getExt();
+            System.out.println(cmd);
+            Process process = Runtime.getRuntime().exec(cmd);
+            BufferedReader in = new BufferedReader(new InputStreamReader(process.getErrorStream()));
+            String line;
+            while ((line = in.readLine()) != null) {
+                System.out.println(line);
```

Comment:
System out println ?

Reply:
for the moment it is necessary

Reply:
For the time being, you can try removing the System.out.println statement but not the while.
For example:
while ((line = in.readLine()) != null) {}

Reply:
when you change it I suggest use do while instead of while according your logic
do {
    lien = in.readLine();
} while (line !=null);

Reply:
I finished trying the code with do-while and it works fine, thanks Paolo for your suggestion